Modern/wrapperPlugins/src/main/java/org/bonej/wrapperPlugins/IntertrabecularAngleWrapper.java

```diff
@@ -88,7 +88,7 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * @see sc.fiji.analyzeSkeleton.AnalyzeSkeleton_
  */
 
-@Plugin(type = Command.class, menuPath = "Plugins>BoneJ>Inter-trabecular Angles")
+@Plugin(type = Command.class, menuPath = "Plugins>BoneJ>Intertrabecular angles")
 public class IntertrabecularAngleWrapper extends ContextCommand {
```

Comment:
In the original publication the name is "Inter-trabecular Angle" (singular, spelt with a dash) - maybe should be kept that way for consistency? I don't mind capitalization, whatever other plugin menu paths do.